hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/SummarySketch.java

```diff
@@ -31,7 +38,9 @@ public TableSummary() {
         public TableSummary add(TableSummary other) {
             Schema s = this.schema;
             if (this.schema == null)
```

Comment:
Is it possible to have a Zero value in that case rather than null? You've used that style elsewhere and I think it is cleaner.